### DISCUSSION TITLE
Image is influenced by Opacity of outter Button and Border

### DISCUSSION THREAD

**Discussion by desertscott** [AvaloniaUI/Avalonia#8814] [General]
![camera view 2022-08-23 111107](https://user-images.githubusercontent.com/90024341/186061765-b0f5c53e-24f3-4d0c-9c55-dc77df1f6a8d.png)
Four buttons have a parent Border respectively, and have a white Image as Child. Opacity of Border and Button is 0.5, which change the color of Image. The default opacity of Image is 1, if change it to 4 or higher, it will display white color.

**Comment by timunie** [comment]
Yes, Opacity will be inherited by the children. So your Border has `0.5` and your Button also has `0.5`. That will be `0.5*0.5 = 0.25 = 25 %`. So if you then multiply by `4`, you have `1` again. 

If you want to have the opacity only applied to the Background, I'd use a semi-transparent brush instead.

**Reply by desertscott** [reply]
Thanks, get it.
